examples/to_ssa.py

```diff
@@ -74,6 +74,9 @@ def _rename(block):
             for p in phis[s]:
                 if stack[p]:
                     phi_args[s][p].append((block, stack[p][0]))
+                else:
+                    # The variable is not defined on this path
+                    phi_args[s][p].append((block, "__undefined"))
```

Comment:
This isn't 100% necessary, as `brili` supports `phi` nodes with incomplete sets of labels. If the label isn't present it treats the variable as undefined. However, the bril spec says that "It is an error to use a phi instruction when [..] the instruction does not contain an entry for the second-most-recently-executed label".

Let me know what you would prefer.

Reply:
This is perfect! I can definitely see it going either way, but I like that this version of the pass, at least, maintains the invariant that phi-nodes reflect the in-degree of the CFG node they are attached to (as they do in LLVM, for example).